### PR TITLE
Add location of auto auth setting to ecomm documentation.

### PR DESCRIPTION
**Pull request by vkaracic**
Adding a bit more info about how to enable auto auth on LMS. @srpearce thoughts? Perhaps a review? :)

### CODE REVIEW

File: loadtests/ecommerce/README.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Getting Started
...
+In order to run the tests, auto auth must be enabled on the LMS, which can be found in ``lms.env.json`` under ``AUTOMATIC_AUTH_FOR_TESTING``. In addition, courses to be used for testing must be configured on both the LMS and Otto. At a minimum, these load tests require the existence of two courses on the LMS instance being used for testing. The edX Demonstration Course should be present by default on most LMS instances. Use Studio to create a course. Use Otto’s "Course Administration Tool" (CAT) to finish configuring the courses you created above. When developing, you can find the CAT at ``http://localhost:8002/courses/``. Add both of the courses present on your LMS instance to Otto. Configure one as a "Free (Honor)" course, and the second as a "Verified" course.
```
Comment (srpearce) [inline loadtests/ecommerce/README.rst]: For readability, I would suggest splitting this paragraph out into steps—making sure that any prerequisites are either mentioned before the steps start, or included in the steps themselves. 

Example 1:

Load tests are configured using a YAML file. Courses to be used for testing must be configured on both the LMS and Otto. At a minimum, these load tests require the existence of two courses on the LMS instance being used for testing. 

1. Create a YAML file and name it settings_files/ecommerce.yml. Use the settings_files/ecommerce.yml.example file as an example. Git will ignore this file, so you can put secrets in it.
2. On the LMS, enable automatic authentication. To do this, open the lms.env.json file and locate AUTOMATIC_AUTH_FOR_TESTING.
3. Use Studio to create one or more courses. The edX Demonstration Course is present by default on most LMS instances. If the edX Demonstration Course is present, create one course. If the instance does not include the edX Demonstration Course, create two courses.
4. Use Otto’s "Course Administration Tool" (CAT) to finish configuring the course or courses you created above. You can find the CAT at ``http://localhost:8002/courses/``. 
5. Add both of the courses present on your LMS instance to Otto. Configure one as a "Free (Honor)" course, and the second as a "Verified" course.

Example 2:

Load tests are configured using a YAML file. Refer to settings_files/ecommerce.yml.example for an example, then create your own alongside it at settings_files/ecommerce.yml. Git will ignore this file, so you can put secrets in it. In order to run the tests, auto auth must be enabled on the LMS, which can be found in ``lms.env.json`` under ``AUTOMATIC_AUTH_FOR_TESTING``. In addition, courses to be used for testing must be configured on both the LMS and Otto. At a minimum, these load tests require the existence of two courses on the LMS instance being used for testing. The edX Demonstration Course should be present by default on most LMS instances. 

1. Use Studio to create a course. 
2. Use Otto’s "Course Administration Tool" (CAT) to finish configuring the courses you created above. When developing, you can find the CAT at ``http://localhost:8002/courses/``. 
3. Add both of the courses present on your LMS instance to Otto. Configure one as a "Free (Honor)" course, and the second as a "Verified" course.

**Comment by vkaracic** [comment]
@srpearce I went with your second suggestion. Let me know if this looks good.

**Comment by srpearce** [comment]
👍

**Comment by pwnage101** [comment]
just looked at this.  Perhaps you would like to poke @edx/ecommerce ?

**Pull request merged**